### PR TITLE
enable universal wheels

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
@aanand @shin- ?

File: setup.py [hunk 1]
```diff
@@ -48,7 +50,25 @@ def find_version(*file_paths):
...
+            if key.startswith(':') and pkg_resources.evaluate_marker(key[1:]):
+                install_requires.extend(value)
+except Exception:
+    logging.getLogger(__name__).exception(
+        'Something went wrong calculating platform specific dependencies, so '
+        "you're getting them all!"
+    )
+    for key, value in extras_require.items():
+        if key.startswith(':'):
+            install_requires.extend(value)
```
Comment (dnephin) [inline setup.py]: What's this about?

Comment (graingert) [inline setup.py]: this lets you support extras_require on old setuptools

Comment (graingert) [inline setup.py]: @kevin1024 and I worked this out over a few iterations of users with odd Python installs https://github.com/kevin1024/vcrpy/blob/master/setup.py#L37-L49

**Comment by graingert** [comment]
@aanand @shin- any movement on this?

**Comment by graingert** [comment]
@aanand @shin- any movement on this, it's been over a week

**Comment by graingert** [comment]
this will result in much faster installs, with less downloading for users of docker-compose from PyPI

**Comment by graingert** [comment]
@aanand @shin- any movement on this?

**Comment by shin-** [comment]
Sorry, I was away for a couple weeks. I'll take a look asap.

**Pull request merged**